src/containers/Network/UpgradeStatus.tsx

```diff
@@ -111,37 +115,31 @@ export const UpgradeStatus = () => {
   return (
     <div className="network-page">
       <Streams validators={vList} updateValidators={updateValidators} />
-      {validatorCount && (
+      {
         // @ts-ignore - Work around for complex type assignment issues
         <Hexagons data={validations} list={vList} />
```

Comment:
What's the type difficulty here?

Reply:
There is weirdness with `connect` where typescript interprets the component as having no props